llvm/utils/TableGen/GlobalISelMatchTable.cpp

```diff
@@ -243,7 +330,7 @@ void MatchTable::emitDeclaration(raw_ostream &OS) const {
     if (I->Flags & MatchTableRecord::MTRF_Outdent)
       Indentation -= 2;
   }
-  OS << "};\n";
+  OS << "}; // " << CurrentSize << " bytes\n";
```

Comment:
Can this be commited seperately so that we can see the bytes reduction?
It can be `CurrentSize * 8` bytes without your changes.

Reply:
Is it alright if I just git push that straight away or do I need to do another PR?

Reply:
Please create another PR for it and add some test coverages (you can just add some CHECKs to check the table size).